fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/RepositoryPaths.java

```diff
@@ -126,6 +126,26 @@ public static String getVocabulariesBase() {
         return vocabulariesBase;
     }
 
+    /**
+     * Prepend id with defined levels of hashed containers based on the values.
+     * For example, 9bd8b60e-93a2-4b66-8f0a-b62338483b39 would become
+     *    9b/d8/b6/9bd8b60e-93a2-4b66-8f0a-b62338483b39
+     *
+     * @param id
```

Comment:
would someone mind filling out the javadoc here to make it more useful?

Reply:
fixed